.github/actions/build/action.yml

```
@@ -12,6 +12,7 @@ runs:
       # ensure that the installed compiler version is used
       run: |
         mkdir build && cd build
+        export CMAKE_PREFIX_PATH=/opt/serial/kokkos/lib/cmake/Kokkos/
```

```suggestion
        export CMAKE_PREFIX_PATH=/opt/openmp
```
I think GMG usually wants to test OpenMP